oryx-app/src/test/java/com/cloudera/oryx/app/speed/kmeans/KMeansSpeedIT.java

```diff
@@ -46,7 +46,7 @@
   public void testKMeansSpeed() throws Exception {
     Map<String,Object> overlayConfig = new HashMap<>();
     overlayConfig.put("oryx.speed.model-manager-class", KMeansSpeedModelManager.class.getName());
-    overlayConfig.put("oryx.speed.streaming.generation-interval-sec", 10);
+    overlayConfig.put("oryx.speed.streaming.generation-interval-sec", 30);
```

Comment:
That's fine. Before it was waiting 10 seconds for data that should be sent in 3 seconds. Now it's 30 sec for 2.1 sec of data. It doesn't make sense that this is actually the problem but I'm not sure what is.